### PR TITLE
Automate release

### PR DESCRIPTION
**Pull request by lievenhey**
This PR adds an automatic latest release aswell as release-please which provides a pr that can be merged to create a new release.

closes: #605

### CODE REVIEW

File: .github/workflows/release-appimage.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      with:
+        fetch-depth: 0
+        submodules: recursive
+
+    - name: Build
+      uses: docker://ghcr.io/kdab/kdesrc-build:latest
+
+    - name: Delete latest release
+      run: |
+        gh release delete latest --cleanup-tag || true
```
Comment (milianw) [inline .github/workflows/release-appimage.yml:24]: is "latest" the name here? can we call it "continuous" to revive the existing old continuous release from back then?

Comment (lievenhey) [reply .github/workflows/release-appimage.yml:24]: yes that is possible

**Review by milianw — APPROVED** [review]
lgtm, thank you

**Comment by GitMensch** [comment]
I currently fail to understand that - the idea is that for each commit on master there is a nightly release, no?
Then I neither see where this defined nor where the nightly release is that would solve the now closed #605.

**Comment by lievenhey** [comment]
Done, seems like ether `ncipollo/release-action` or github automatically add a `draft` tag. Now I use the github cli to remove that tag.

I don't think changing the name to `hotspot-continuous.AppImage` is a good idea. The current name is a perfect description of the commit that was used to build the appimage. If we get a bug report we can then use the name to identify which exact version was used. Maybe the bug is already fixed but the user didn't update, etc ...

**Comment by GitMensch** [comment]
`./hotspot... --version` has the output, as well as the version info in the GUI.
The idea here is to get a _stable_ download URL, and for that we need - additional to the updated release - an identical name.

If you still want the versioned name: what do you think about having the appimage twice (once with a stable name)?

**Pull request merged**